edenai_apis/apis/iointelligence/iointelligence_api.py

```
@@ -0,0 +1,104 @@
+from edenai_apis.features.llm.chat.chat_dataclass import (
+    ChatDataClass,
+    StreamChat as StreamChatCompletion,
+)
+from edenai_apis.utils.exception import ProviderException
+from edenai_apis.llmengine.types.response_types import ResponseModel
+
+
+class IointelligenceApi(ProviderInterface, LlmInterface, TextInterface):
+    provider_name = "iointelligence"
+
+    def __init__(self, api_keys: Dict = {}) -> None:
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)